llvm/include/llvm/CAS/CachingOnDiskFileSystem.h

```diff
@@ -14,10 +14,12 @@
 #include "llvm/Support/Error.h"
 #include "llvm/Support/VirtualFileSystem.h"
 #include <map>
+#include <memory>
 
 namespace llvm {
 namespace cas {
 
+class ActionCache;
```

Comment:
Looks like there are a couple left-over changes in this file (and the .cpp as well).